app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -23,216 +25,248 @@ class RoomProfilePage extends ConsumerWidget {
+                    style: Theme.of(context).textTheme.titleMedium,
                   ),
-                ],
+                  error: (err, stackTrace) {
+                    debugPrint('Some error occured $err');
+                    return Text(
+                      roomId,
+                      overflow: TextOverflow.clip,
+                      style: Theme.of(context).textTheme.titleSmall,
+                    );
+                  },
+                  loading: () => const CircularProgressIndicator(),
+                ),
+              ],
+            ),
+          ),
+          SliverPadding(
+            padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 8),
+            sliver: convo.when(
+              data: (data) => SliverToBoxAdapter(
+                child: RenderHtml(
+                  text: data.topic() ?? '',
+                  defaultTextStyle: Theme.of(context).textTheme.bodySmall,
+                ),
               ),
+              loading: () => const Text('loading...'),
+              error: (e, s) => Text('Error: $e'),
             ),
-            SliverPadding(
-              padding: const EdgeInsets.all(8.0),
-              sliver: SliverToBoxAdapter(
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  children: [
-                    Padding(
-                      padding: const EdgeInsets.symmetric(horizontal: 8),
-                      child: Text(
-                        roomId,
-                        style: Theme.of(context).textTheme.bodyMedium!.copyWith(
-                              color: Theme.of(context).colorScheme.neutral5,
-                            ),
-                        softWrap: true,
+          ),
+          SliverToBoxAdapter(
+            child: SettingsList(
+              physics: const NeverScrollableScrollPhysics(),
+              darkTheme: SettingsThemeData(
+                settingsListBackground: Colors.transparent,
+                settingsSectionBackground:
+                    Theme.of(context).colorScheme.onPrimary,
+                dividerColor: Colors.transparent,
+                leadingIconsColor: Theme.of(context).colorScheme.neutral6,
+              ),
+              shrinkWrap: true,
+              sections: [
+                SettingsSection(
+                  tiles: [
+                    myMembership.when(
+                      data: (membership) => SettingsTile.navigation(
+                        onPressed: (ctx) {
+                          membership!.canString('CanInvite')
+                              ? ctx.pushNamed(
+                                  Routes.spaceInvite.name,
+                                  pathParameters: {'spaceId': roomId},
+                                )
+                              : customMsgSnackbar(
+                                  ctx,
+                                  'Not enough power level for invites, ask room administrator to change it',
```

Comment:
eh... the invite button should rather not be shown at all if the user doesn't have the power level to do so ...

but I am okay if you put that into a ticket post-merge..